unity-renderer/Assets/DCLPlugins/ECS7/ECSComponents/ComponentID/ComponentID.cs

```diff
@@ -10,10 +10,11 @@ public static class ComponentID
         public const int CYLINDER_SHAPE = 1016;
         public const int G_L_T_F_SHAPE = 1040;
         public const int N_F_T_SHAPE = 1040;
```

Comment:
This is an autogenerated file, it uses the the name of the proto to generate the underscore, since GLTF and NFT are acronyms , they are separated by underscore, seems like in one version of ecs their had the same number

Reply:
I have change the way this file is generated to match